xhype/xhype/LICENSE

```diff
@@ -0,0 +1,340 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
I am wondering why you need this here, since there is one at top level?

Reply:
It is not necessary. I can remove it since there is one at top level.